config/equinix.spc

```diff
@@ -2,6 +2,7 @@ connection "equinix" {
   plugin = "equinix"
 
   # API Token for your Equinix Metal account
-  # See https://metal.equinix.com/developers/docs/accounts/users/#api-keys
-  #token = "YOUR_EQUINIX_METAL_API_TOKEN"
+  # Reference: https://metal.equinix.com/developers/docs/accounts/users/
```

Comment:
Would it be more helpful if we link to the API section in particular? https://metal.equinix.com/developers/docs/accounts/users/#api-keys